src/main/java/net/coderbot/iris/shadow/ShadowMatrices.java

```diff
@@ -23,6 +23,21 @@
 		};
 	}
 
+	public static float[] createPerspectiveMatrix(float fov, float aspect) {
+		float i = (float)(1.0D / Math.tan(fov * 0.01745329238474369D / 2.0D));
+		Matrix4f matrix4f = new Matrix4f();
+		return new float[] {
```

Comment:
This code gives me the feeling that it was copied from somewhere, with the arbitrary constants and the unused Matrix4f. I'd prefer if this code weren't copied & had more documentation, such as what units FOV has.

Reply:
Correct, it was copied from Vanilla's projection method. I'll look into this.